docs/docs/guides/tools/otel.md

```diff
@@ -0,0 +1,175 @@
+# Send OpenTelemetry Traces to Weave 
+
+## Overview
+You can use the OTEL Trace API to send OpenTelemetry-compatible trace data to Weave. This allows you to integrate observability from any OTEL-instrumented application—like GenAI apps, backend services, or external APIs—into Weave's ecosystem.
+
+Once ingested, trace data can be used to visualize, compare, and correlate with other Weave data.
+
+**Use Cases:**
+- Monitor OpenAI calls alongside frontend or backend performance
+- Trace long-running chains and identify bottlenecks
+- Compare traces across deployments or versions
+
+## Get started
+1. Get your API key from [wandb.ai/authorize](https://wandb.ai/authorize).
+2. Define your `project_id` and OTEL endpoint.
+3. Ensure that you have write access to the project.
+4. Review the [API endpoint details](#api-endpoint-details).
+5. Generate the [required authorization headers](#generate-authorization-headers) and set up your OTLP HTTP exporter using the provided values.
+6. Try an example use case.
+
+## API endpoint details
+- **URL**: `https://api.wandb.ai/otel/v1/traces`
```

Comment:
Isn't this URL on the trace server?

Reply:
Yeah that's the wrong URL, I didn't notice this until too late. I'll open something to fix it